uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -214,6 +218,7 @@
 ############### Run runTarchiveLoader###########################
 ################################################################
 $output = $imaging_upload->runTarchiveLoader();
+$imaging_upload->updateMRIUploadTable('Processing', 0);
```

Comment:
Because at this point, processED is set to 1 in tarchiveLoader, so processING should be set back to 0